Resources/Prototypes/Catalog/VendingMachines/vending-machine-restock-pack.yml

```diff
@@ -0,0 +1,25 @@
+- type: entity
+  id: VendingMachineColaRestock
```

Comment:
once I determine if restock packs should contain actual items or not, I'll extract a proper base prototype